frontend/src/components/navigation/NavigationStyled.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const FooterWrapper = styled.nav`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  height: 10vh;
+  bottom: 0;
+  position: absolute;
+  background-color: #f1f1f1;
+`;
```

Comment:
I would always set the "position:absolute" as the first attribute. So whenever you want to relate another element to this styled component, you see quickly what criteria you used here.